cmd/kind/load/docker-image/docker-image.go

```diff
@@ -68,6 +70,13 @@ func NewCommand() *cobra.Command {
 }
 
 func runE(flags *flagpole, cmd *cobra.Command, args []string) error {
+	imageName := args[0]
+	// Check that the image exists locally, if not return error
+	// {{.Os}} seems to be an attribute specific to images only
```

Comment:
seems a bit brittle, I wonder if there's a better option... 🤔

Reply:
https://docs.docker.com/engine/reference/commandline/image_inspect/ ?

Reply:
It should be possible to use `docker image inspect` or `docker image ls`. In terms of meaning, `docker image ls` should be more appropriate.

Reply:
`docker image ls` doesn´t discriminate by tag, I find more appropriate the behavior of `image inspect` that only returns the matching image with the tag included

Reply:
> `docker image ls` doesn´t discriminate by tag,

It can be distinguished by tag. i.e.

```
(MoeLove) ➜  ~ docker image ls debian
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
debian              latest              8d31923452f8        5 weeks ago         101MB
debian              stretch             de8b49d4b0b3        5 months ago        101MB
(MoeLove) ➜  ~ docker image ls debian:stretch
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
debian              stretch             de8b49d4b0b3        5 months ago        101MB
(MoeLove) ➜  ~ docker image ls debian:latest 
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
debian              latest              8d31923452f8        5 weeks ago         101MB
```

or

```
(MoeLove) ➜  ~ docker image ls -f reference=debian
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
debian              latest              8d31923452f8        5 weeks ago         101MB
debian              stretch             de8b49d4b0b3        5 months ago        101MB
(MoeLove) ➜  ~ docker image ls -f reference=debian:stretch
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
debian              stretch             de8b49d4b0b3        5 months ago        101MB
(MoeLove) ➜  ~ docker image ls -f reference=debian:latest 
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
debian              latest              8d31923452f8        5 weeks ago         101MB

```

Reply:
@tao12345666333 my bad 😅 ,  I didn't explain it correctly, my point was that the user should expect to load only one image, and use ` latest` in absence of any tag, similar to another docker commands

``` 
docker image ls nginx
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
nginx               1.17-alpine         bfba26ca350c        7 days ago          20.5MB
nginx               1.15.12-alpine      0be75340bd9b        8 weeks ago         16.1MB
nginx               1.14-alpine         cafef9fe265b        3 months ago        16MB
```